src/Reflection/ClassReflection.php

```diff
@@ -113,65 +108,55 @@ public function getMethod($name)
     }
 
     /**
-     * Get reflection objects of all methods
+     * {@inheritDoc}
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf